lib/PseudoFactory.js

```diff
@@ -162,8 +164,8 @@ var PseudoFactory = function(options) {
             });
             break;
 
-        case 'debug-asian':
```

Comment:
debug-asian should still be supported as an undocumented alias for debug-han-simplified for backwards compatibility. That way, people do not need to change their project.json for this to continue working.

Reply:
I added to code to map `debug-asin` to `debug-han-simplified`.
https://github.com/iLib-js/loctool/blob/pseudostyle/lib/PseudoFactory.js#L96
It would not break the current `debug-asian` usage case. Isn't it enough?

Reply:
Oh, I see. Okay, approved